src/@polymer/polymer-lit-element.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * @license
+ * Copyright (c) 2017 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at
+ * http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+import { PropertiesMixin } from '../../@polymer/polymer/lib/mixins/properties-mixin.js';
+import { TemplateResult } from '../../lit-html/lit-html.js';
+import { render } from '../../lit-html/lib/lit-extended.js';
+
+export { html } from '../../lit-html/lit-html.js';
+export class PolymerLitElement extends PropertiesMixin(HTMLElement) {
+
+  ready() {
+    this.attachShadow({mode: 'open'});
```

Comment:
Given this line, I think we can add a `shadowRoot: ShadowRoot` as property on this class. This should remove the need for `(el.shadowRoot as ShadowRoot)` in every usage of this field.